tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -742,3 +742,88 @@ describe("Test Application deployment based on 'clusterSelector'", { tags: '@p1'
     )
   })
 });
+
+describe('Test with disablePolling', { tags: '@p1'}, () => {
+  
+  beforeEach("Ensuring Github repo has desired amount of replicas (2)", () => {
+
+    const gh_private_pwd = Cypress.env("gh_private_pwd")
+
+    cy.exec(
+      `
+      echo -e "Cloning repo"
+      git clone https://fleetqa:${gh_private_pwd}@github.com/fleetqa/fleet-qa-examples-public.git --branch=main $PWD/fleet-qa-examples-public
+
+      echo -e "Confirming repo exist in path"
+      test -e "$PWD/fleet-qa-examples-public/disable-polling"
+      if [ $? -ne 0 ]; then
+        echo "Error: path $PWD/fleet-qa-examples-public/disable-polling does not exist"
+        exit 1
+      fi
+
+      echo -e "Ensuring replicas is 2"
+      sed -i 's/replicas: ..*/replicas: 2/g' $PWD/fleet-qa-examples-public/disable-polling/nginx.yaml
+
+      echo -e "Commiting and pushing if needed "
```

Comment:
Why `pushing if needed`  please elaborate more in the same `echo`?

Reply:
By 'if needed' I meant if number of replicas is different from initially pre-established.
Ok, let me rephrase that.
Thanks.

Reply:
Done, thanks a lot for the suggestion